metrics/grafana/tikv_details.dashboard.py

```diff
@@ -1075,6 +1075,22 @@ def gRPC() -> RowPanel:
                     ),
                 ],
             ),
+            graph_panel(
+                title=r"gRPC batch commands wait duration",
+                description=r"The 99.99% wait time of gRPC batch commands",
+                yaxes=yaxes(left_format=UNITS.SECONDS, log_base=2),
+                targets=[
+                    target(
+                        expr=expr_histogram_quantile(
+                            0.9999,
+                            "tikv_grpc_batch_commands_wait_duration_seconds",
+                            is_optional_quantile=False,
+                        ),
+                        legend_format="max-wait",
```

Comment:
```suggestion
                        legend_format="P9999",
```
the quantile of max-wait should be P100

Reply:
Make sense, addressed